tests/fast-integration/package-lock.json

```diff
@@ -1,4446 +1,8 @@
 {
   "name": "kyma-js",
   "version": "0.0.1-alpha.12",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
I think we should stay with the second version of it. Did you try to upgrade `node.js` on your machine?